pages/how_tos/bitbucket.md.erb

```diff
@@ -1,37 +1,44 @@
 # Bitbucket Repo Access
```

Comment:
The other "How To" titles are just "GitLab" and "GitLab Enterprise". Does that mean this one should just be "Bitbucket"?